pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1931,33 +2077,37 @@ public class ServiceConfiguration implements PulsarConfiguration {
     private int managedLedgerCursorRolloverTimeInSeconds = 14400;
     @FieldContext(
         category = CATEGORY_STORAGE_ML,
-        doc = "Max number of `acknowledgment holes` that are going to be persistently stored.\n\n"
-            + "When acknowledging out of order, a consumer will leave holes that are supposed"
-            + " to be quickly filled by acking all the messages. The information of which"
-            + " messages are acknowledged is persisted by compressing in `ranges` of messages"
-            + " that were acknowledged. After the max number of ranges is reached, the information"
-            + " will only be tracked in memory and messages will be redelivered in case of"
-            + " crashes.")
+            doc = """
+                Max number of `acknowledgment holes` that are going to be persistently stored.
+
+                When acknowledging out of order, a consumer will leave holes that are supposed\
+                 to be quickly filled by acking all the messages. The information of which\
```

Comment:
Better put the fist space at the end of previous line.

Reply:
Ok. I will improve it.